### DISCUSSION TITLE
Postituslistasta luopuminen, GitHub-keskustelut tilalle / Mailing lists replaced by GitHub discussions

### DISCUSSION THREAD

**Discussion by hatapitk** [voikko/discussion#1] [Announcements]
Hei kaikille! / Hello everyone!

Pitkään hyvin palvelleen postituslistapalvelimemme Debian-käyttöjärjestelmän päivitys huhtikuussa johti siihen, että käytössä ollut Mailman 2 olisi pitänyt päivittää Mailmanin 3-versioon. Tämä olisi vaatinut työtä, ja sähköpostipalvelimen ylläpitäminen harrastusprojektina on muutenkin nykymaailmassa jo melko haastavaa tiukentuneiden roskapostisuodatusten sekä lainsäädännön takia.

Päädyin siksi siihen, että en enää palauta postituslistaa käyttöön, vaan siirrämme keskustelun tänne Github-alustalle. Pahoittelen, etten tajunnut tiedottaa asiasta ennen päivitystä. Ehkä keksin vielä tavan tiedottaa vanhoja tilaajia asiasta jotain kautta. Ymmärrän myös, että tämä alusta ei ole ehkä kaikille mieluinen toimia, mutta vanha postituslistakin oli jo melko hiljainen. Ehkä tämä foorumi saattaa jopa osoittautua toimivammaksi kuin aiempi.

Tämän keskustelukanava on tarkoitettu avoimeksi kaikille kiinnostuneille. Vanhaan tapaan keskustelua voi käydä joko suomeksi tai englanniksi (ei enää erillisiä listoja näille). Jos huomaatte, että jokin asetus on pielessä ettekä pysty osallistumaan keskusteluun, voitte laittaa minulle myös suoraan sähköpostia (hatapitk@iki.fi). Tervetuloa mukaan!

---

The update of the Debian operating system of our long-serving mailing list server in April meant that the Mailman 2 in use should have been updated to the Mailman 3 version. This would have required work, and maintaining an e-mail server as a hobby project is already quite challenging in today's world due to stricter spam filters and legislation.

That's why I ended up not reactivating the mailing list anymore, but we'll move the discussion here to the Github platform. I'm sorry I didn't realize to inform about it before the update. Maybe I'll figure out a way to inform old subscribers about it. I also understand that this platform may not please everyone, but the old mailing list was already quite quiet. Maybe this forum might even turn out to be more functional than the previous one.

This discussion channel is intended to be open to anyone interested. As before you can post in English or Finnish but we don't have separate forums for these anymore. If you notice that some setting is wrong and you cannot participate in the discussion, you can also send me an email directly (hatapitk@iki.fi). Welcome aboard!

**Comment by tlikonen** [comment]
Harri Pitkänen kirjoitti 12.5.2023 kello 16.32:

> Päädyin siksi siihen, että en enää palauta postituslistaa käyttöön,
> vaan siirrämme keskustelun tänne Github-alustalle. Pahoittelen, etten
> tajunnut tiedottaa asiasta ennen päivitystä. Ehkä keksin vielä tavan
> tiedottaa vanhoja tilaajia asiasta jotain kautta. Ymmärrän myös, että
> tämä alusta ei ole ehkä kaikille mieluinen toimia, mutta vanha
> postituslistakin oli jo melko hiljainen. Ehkä tämä foorumi saattaa
> jopa osoittautua toimivammaksi kuin aiempi.

Kiitokset. Tämäkin sopii minulle ihan mainiosti. Piti Joukahaisesta
kohta kysyäkin, mutta sain siihen jo vastauksen.

(Samalla testaan, kuinka viestiin voi vastata sähköpostiohjelmalla.
Oletettavasti toimii...)

**Comment by hatapitk** [comment]
Näyttää toimivan, hieno juttu!

pe 12. toukok. 2023 klo 17.24 Teemu Likonen ***@***.***>
kirjoitti:

**Comment by ahomansikka** [comment]
Ööööh? Miten voi aloittaa uuden aiheen? Mutta:


Päivitin Sukijan Solr:n versioon 9.2.1.

Imurointi: git clone https://github.com/ahomansikka/sukija.git

Päivitys: git pull
